oidc-ui/public/locales/ar.json

```diff
@@ -74,12 +74,15 @@
   "LoginQRCode": {
     "sign_in_with_inji": "تسجيل الدخول باستخدام Inji",
     "scan_with_inji": "امسح باستخدام تطبيق Inji لتسجيل الدخول",
+    "scan_with_wallet": "امسح باستخدام تطبيق {{walletName}} لتسجيل الدخول",
     "inji_app": "تطبيق إنجي",
     "dont_have_inji": "ليس لديك إنجي؟",
+    "dont_have_wallet": "؟{{walletName}} ليس لديك",
```

Comment:
Will require multi-lang support for wallet name

Reply:
we have to create a separate task for this issue

Reply:
Task created https://mosip.atlassian.net/browse/ES-279